src/Components/Comments.jsx

```diff
@@ -1,39 +1,104 @@
-import React from "react";
-import AddComment from "./AddComment";
+import React, { useRef, useState, useEffect } from "react";
+import getUsers from "../Utilis/FetchData";
+import { useLocation } from "react-router-dom";
+import postComment from "../Utilis/PostComment";
+import { logDOM } from "@testing-library/react";
 
 export default function Comments() {
+ 
+  const location = useLocation();
+ 
+  const { articles } = location.state;
+    
+  const [user, setUser] = useState([]);
+  const inputRef = useRef();
+
+  useEffect(() => {
+    
+    getUsers()
+      .then((data) => {
+        for (let i in data) {
+          if (data[i].username === articles[0].author) 
+          setUser(data[i]);
+        }
+      })
+      .catch((err) => alert(err));
```

Comment:
```js
      .catch((err) => alert(err));
```
Don't put JS errors on the screen for users to see. This might help you as a dev but a user should never see this